Pre-Processing/get_non_soft_story_images.py

```diff
@@ -29,7 +36,7 @@ def get_image(address):
         response = requests.get(url)
         image = response.content
     except:
```

Comment:
Ideally, you never want to use blank `except` statements but instead, you want to [catch the exception](https://docs.python.org/3.8/tutorial/errors.html).